modules/administration/pages/monitoring.adoc

```diff
@@ -325,13 +325,13 @@ For more information on PushProx, see https://github.com/RobustPerception/PushPr
 === Reverse Proxy Setup
 
 Prometheus fetches metrics using a pull mechanism, so the server must be able to establish TCP connections to monitored clients.
```

Comment:
Truth to be told, since we explain "exporters" below, we could very well say here:

> to establish TCP connections to the exporters on the monitored clients.

Reply:
Or even more correct:

> to establish TCP connections to each exporter on the monitored clients.

Reply:
done